app/views/questions/triage_backtaxes/edit.html.erb

```diff
@@ -1,2 +1,3 @@
-<% content_for :form_question, t("views.questions.triage_backtaxes.title") %>
+<% content_for :form_question, t("views.questions.triage_backtaxes.title",
+                                 backtax_years: TaxReturn.backtax_years.reverse.to_sentence(last_word_connector: " #{t("general.and")} ")) %>
```

Comment:
I believe this `last_word_connector` should be `", #{t("general.and")} "`. This would retain the [Oxford comma](https://en.wikipedia.org/wiki/Serial_comma) in the original title.

Before:

```
irb(main):001:0> I18n.t("views.questions.triage_backtaxes.title")
=> "Have you filed taxes for 2017, 2018, and 2019?"
```

After:

```
irb(main):008:0> I18n.t("views.questions.triage_backtaxes.title",
                                 backtax_years: TaxReturn.backtax_years.reverse.to_sentence(last_word_connector: " #{I18n.t("general.and")} "))
=> "Have you filed taxes for 2017, 2018 and 2019?"
```

I strongly encourage this change because the comma was in the original string. Not strictly a blocker.